From 3979ada85b67c02a588bf66308027b70fb882b48 Mon Sep 17 00:00:00 2001
From: Charles-IV <cthefourth@gmail.com>
Date: Thu, 12 Jul 2018 13:14:28 +0100
Subject: [PATCH 223/249] tissot: use custom twrp

official twrp doesn't work well for this device - fastboot boot doesn't
work.

Change-Id: Id9ea643e28f30965b8f90f36b94badc1fadbb119
---
 _data/devices/tissot.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_data/devices/tissot.yml b/_data/devices/tissot.yml
index bb4905c..bbccdd1 100644
--- a/_data/devices/tissot.yml
+++ b/_data/devices/tissot.yml
@@ -12,6 +12,7 @@ cpu: Cortex-A53
 cpu_cores: '8'
 cpu_freq: 2.0 GHz
 current_branch: 15.1
+custom_twrp_link: https://drive.google.com/a/my.smccd.edu/uc?id=1tjfpvJKaEtviITgd83plS9p0BMQYHgEe&export=download
 depth: 7.3 mm (0.29 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
   Keep holding both buttons until the word "FASTBOOT" appears on the screen, then release.
-- 
2.17.1

